start-site/src/main/java/io/spring/start/site/container/DockerService.java

```diff
@@ -33,12 +34,15 @@ public class DockerService implements Consumer<Builder> {
 
 	private final String website;
 
+	private final String command;
+
 	private final int[] ports;
 
-	DockerService(String image, String imageTag, String website, int... ports) {
+	DockerService(String image, String imageTag, String website, String command, int... ports) {
```

Comment:
Overload this instead so that `SimpleDockerServiceResolver` doesn't have to pass in `null` each time?

Reply:
Yeah, I was thinking the same. That or we should maybe introduce a builder if we keep getting optional attributes there.

Reply:
Adding a builder.